tests/unit/aroi/test_aroi_domain_extraction.py

```
@@ -11,83 +14,202 @@
-    """Test AROI domain extraction matches AROIValidator specification."""
-    
+    """Test AROI domain extraction for both v2 and v3 specs.
+
+    `_simple_aroi_parsing` returns a 3-tuple:
+      (domain_or_'none', version_or_None, proof_type_or_None)
+    """
+
     def setUp(self):
         """Set up test instance with minimal mocking."""
-        # Create a minimal Relays instance just to test _simple_aroi_parsing
         with patch.object(Relays, '__init__', lambda x, **kwargs: None):
```

Ruff (0.15.12)

[warning] Unused lambda argument: `x`

(ARG005)